### PR TITLE
[Bugfix] Docker regression using master regression scripts

### PR DESCRIPTION
**Pull request by ganeshgore**
### Motivate of the pull request
- [x] To address an existing issue. If so, please provide a link to the issue. #239 
- [ ] Breaking new feature. If so, please describe details in the description part.

### Describe the technical details
The docker image has OPENFPGA_PATH already setup to the already installed binary, during current regression test it uses that path to run regression script. This commit unset the OPENFPGA_PATH so that new PATH can be setup.

### Which part of the code base require a change
**In general, modification on existing submodules are not acceptable. You should push changes to upstream.**
- [x] Regression test

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -266,6 +266,7 @@ jobs:
           bash .github/workflows/install_dependencies_run.sh
           ${PYTHON_EXEC} -m pip install -r requirements.txt
           rsync -am --exclude='openfpga_flow/**' /opt/openfpga/. .
+          unset OPENFPGA_PATH
```
Comment (tangxifan) [inline .github/workflows/build.yml:269]: Do you need to add the unset to LINE 233?

**Review by tangxifan — COMMENTED** [review]
Not sure why this can make the regression tests to be false green.  In Tarachand's PR, there is no code changes in the binary. The changes are in the python scripts which will not be compiled to binary. If it is due to OpenFPGA path, why the regression tests were green?

**Comment by ganeshgore** [comment]
I tried to explain that partially in the pull message.
In master Docker image the binaries and `openfpga_flow` already exist in  /opt/openfpga directory
and OPENFPGA_PATH points to `/opt/openfpga` directory.
When we ran `docker_regression_tests` with docker, it ran regression tests from master image.
So I am unsetting the OPENFPGA_PATH, and following `source openfpga.sh` command will set correct PATH
which is `/_w/openfpga/openfpga/.....` where by default github actions clones repo.

File: .github/workflows/build.yml (hunk 1 above)
Comment (ganeshgore) [reply .github/workflows/build.yml:269]: Nope its only while running from the docker image.

**Review by tangxifan — APPROVED** [review]

**Comment by tangxifan** [comment]
A note for the docker image creation, details can be found in the [documentation](https://openfpga.readthedocs.io/en/master/dev_manual/ci_cd_setup/).
The master docker image is used when there are no code changes on the openfpga binaries.
For each PR, the master docker image may be used.

For master branch, docker image will not be used. Regression tests will always be run on the latest code changes. That explains why master branch failed while PR passed.

**Pull request merged**